src/virtualbase.cpp

```diff
@@ -43,8 +43,16 @@ void Virtualbase::applyTwistAndUpdate(const geometry_msgs::Twist& cmd, double dt
     geometry_msgs::Twist twist;
     if(disable_speedcap_ == false){
```

Comment:
Your change only affects the simulator if the speedcap is not disabled.

Reply:
I must have been sleeping still, thanks